demos/browser/server.js

```diff
@@ -76,58 +76,88 @@ function serve(host = '127.0.0.1:8080') {
         if (!requestUrl.query.title || !requestUrl.query.name) {
           respond(response, 400, 'application/json', JSON.stringify({ error: 'Need parameters: title and name' }));
         }
-        let client = getClientForMeeting(meetingTable[requestUrl.query.title]);
+        const meetingIdFormat = /^[a-fA-F0-9]{8}(?:-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$/
+        let meeting = meetingTable[requestUrl.query.title];
+
+        let client = getClientForMeeting(meeting);
+
+        let primaryMeeting = undefined
+        if (requestUrl.query.primaryExternalMeetingId) {
+          primaryMeeting = meetingTable[requestUrl.query.primaryExternalMeetingId]
+          if (primaryMeeting) {
+            console.info(`Retrieved primary meeting ID ${primaryMeeting.Meeting.MeetingId} for external meeting ID ${requestUrl.query.primaryExternalMeetingId}`)
+          } else if (meetingIdFormat.test(requestUrl.query.primaryExternalMeetingId)) {
+            // Just in case, check if we were passed a regular meeting ID instead of an external ID
+            try {
+              primaryMeeting = await client.getMeeting({
+                MeetingId: requestUrl.query.primaryExternalMeetingId
```

Comment:
[For understanding and double checking] For the primary meeting, does this also need to be `MeetingId` or it will be `PrimaryMeetingId`? Tagging @hensmi-amazon for checking once. 

Same would apply for the code in serverless handler.

Reply:
This should be the primary meeting ID, but the field is called `primaryExternalMeetingId`.  The `title` and `primaryExternalMeetingId` previously contained external IDs.  After this change, we still check for it in our meetings table as an external ID, however if it's not there we try to treat it as a meeting ID.

Same for the serverless handler as well.